resteasy/src/main/java/com/cerner/beadledom/resteasy/ResteasyContextListener.java

```diff
@@ -3,14 +3,14 @@
 import com.cerner.beadledom.lifecycle.GuiceLifecycleContainers;
 import com.cerner.beadledom.lifecycle.LifecycleContainer;
 import com.cerner.beadledom.lifecycle.LifecycleInjector;
-import com.google.common.collect.Lists;
 import com.google.inject.Injector;
 import com.google.inject.Module;
+
```

Comment:
There shouldn't be any line breaks for regular imports with the Google style guide.

Reply:
Could you provide a link to the style guide?

Reply:
https://google.github.io/styleguide/javaguide.html

Reply:
https://github.com/cerner/beadledom/commit/75d364f068be5b48f12a864aa2a8bd938fb929b8